setup.py

```diff
@@ -0,0 +1,26 @@
+import os
+import os.path
+from setuptools import setup
+
+import versioneer
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+packages = []
+for d, _, _ in os.walk(os.path.join(here, 'nc_time_axis')):
+    if os.path.exists(os.path.join(d, '__init__.py')):
+        packages.append(d[len(here)+1:].replace(os.path.sep, '.'))
+
+setup_args = dict(
+    name='nc-time-axis',
+    version=versioneer.get_version(),
+    cmdclass=versioneer.get_cmdclass(),
+    description='netcdftime support for matplotlib axis',
+    author='Philip Elson',
```

Comment:
No :)

I won't take ownership of something you've done.

I was considering putting the author as 'UK Met Office' as in [iris-grib](https://github.com/SciTools/iris-grib/blob/master/setup.py#L59) but I'm not sure of the impacts of that down the line (i.e. if this was to be added to netcdftime's library or matplotlib or dateutil (which is where I think netcdftime.datetime should be)).